### DISCUSSION TITLE
Login via local IP - getting "invalid csrf token" when using Plex Account

### DISCUSSION THREAD

**Discussion by shanelord01** [sct/overseerr#1026] [Help] [answered]
Hi,

I have Overseerr setup behind a proxy, using format https://overseerr.domainname.com:port

I have that domain in the "Application URL" field in settings.

When I login via that domain I can login with my Plex account without issue. When I login via the local IP address of the server, I select "Sign In" and get the Plex window stating "Overseerr would like to sign in to your Plex account" and it shows the correct local IP and my plex account email address, but when I hit "Sign In" on this page, it goes back to Overseerr login page with the error:

"invalid csrf token"

This has previously worked, but I cannot speak to which version as I use ouroboros to auto update. I do have "Enable CSRF Protection" enabled and will try this disabled, but if this is the cause, is there a way to keep this enabled and still have the local IP work?

Anyone else experience this and have a fix?

**UPDATE**: It is caused by having "Enable CSRF Protection" enabled. Any way to keep this protection but still allow local IP login using Plex Account?

Thanks,
Shane.

**Comment by sct** [comment, marked as answer]
No there is no way. CSRF protection requires SSL to work. If you are not behind SSL locally, you will not be able to sign in.
